debezium-bom/pom.xml

```diff
@@ -509,6 +512,13 @@
                 <version>${version.azure.blob}</version>
             </dependency>
 
+            <!-- K8s Config Map Storage -->
+            <dependency>
+                <groupId>io.fabric8</groupId>
+                <artifactId>kubernetes-client</artifactId>
+                <version>${version.kubernetes.client}</version>
+            </dependency>
```

Comment:
This will need to be excluded in DS as we should use the same version as Quarkus kubernetes-client extension. 

I would actually consider putting the entire module under DS project. However on the oposite side it might also be useful for use with engine...

Reply:
We've already had a debate with @mfvitale about it and at the end we opted for the core module.